packages/airnode-validator/test/validator.test.ts

```diff
@@ -1,10 +1,24 @@
 import * as fs from 'fs';
 import path from 'path';
-import { validateWithTemplate } from '../src';
+import { validateWithTemplate, validateJsonWithTemplate } from '../src';
 
 const tests = fs.readdirSync(path.resolve(__dirname, 'validatorTests'));
 const validOutput = { valid: true, messages: [] };
 
+const secretsEnv = {
+  PROVIDER_URL: 'http://127.0.0.1:8545',
+  AIRNODE_RRP_ADDRESS: '0x0000000000000000000000000000000000000000',
+  AIRNODE_WALLET_MNEMONIC: 'achieve climb couple wait accident symbol spy blouse reduce foil echo label',
+  SS_CURRENCY_CONVERTER_API_KEY: '<enter your API key>',
+  CMC_PRO_API_KEY: '<enter your API key>',
+  HTTP_GATEWAY_API_KEY: 'apikey_example_apikey_example_apikey_example',
+  HEARTBEAT_API_KEY: '',
```

Comment:
I would suggest using UUID as example values for API keys.

Reply:
Updated in d83d5fe